multiqc/modules/qualimap/QM_RNASeq.py

```diff
@@ -81,7 +86,7 @@ def parse_reports(self):
             if line.startswith("#"):
                 continue
             coverage, count = line.split(None, 1)
-            coverage = int(round(float(coverage)))
+            coverage = int(round(float(coverage.replace(",", "."))))
```

Comment:
I don't quite understand this change... Is it not possible to get something like `123,456.78` here? Does it not also need similar logic to elsewhere?

Reply:
The coverage file doesn't use decimals, so it would be `123456.78`